core/src/pipeline.h

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 Robin Linden <dev@robinlinden.eu>
+
+#pragma once
+
+#include <deque>
+#include <functional>
+
+namespace n_e_s::core {
+
+class Pipeline {
+    using StepT = std::function<void()>;
+
+public:
+    void push(StepT step);
+
```

Comment:
Remove blank lines between the functions. You can leave the one before `private:`.

Reply:
Done